x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/optimizer/LocalPhysicalPlanOptimizer.java

```diff
@@ -153,660 +75,4 @@ protected List<Batch<PhysicalPlan>> rules(boolean optimizeForEsSource) {
     protected List<Batch<PhysicalPlan>> batches() {
```

Comment:
Maybe a personal preference or looking at this code for long enough to start having a habit: I find it more logical to see the `batches()` definition first and then what a batch contains. In other words, I prefer `batches` be defined before `rules`.